translib/transformer/xfmr_intf.go

```diff
@@ -801,6 +830,63 @@ var DbToYang_intf_eth_port_config_xfmr SubTreeXfmrDbToYang = func(inParams XfmrP
 	return err
 }
 
+var Subscribe_intf_eth_port_config_xfmr SubTreeXfmrSubscribe = func(inParams XfmrSubscInParams) (XfmrSubscOutParams, error) {
+	var err error
+	var result XfmrSubscOutParams
+
+	if inParams.subscProc == TRANSLATE_SUBSCRIBE {
+		log.Info("Subscribe_intf_eth_port_config_xfmr: inParams.subscProc: ", inParams.subscProc)
```

Comment:
I feel this log is unnecessary.. You have already checked for the `inParams.subscProc` value and taken a decision. Make it a verbose log if you want to retain